selftests/functional/test_skiptests.py

```diff
@@ -32,15 +33,16 @@ def test2(self):
     def test3(self):
         pass
 
-    @avocado.skipIf(False)
+    @avocado.skipIf(False, "won't be skipped")
     def test4(self):
         pass
 
-    @avocado.skipUnless(True)
+    @avocado.skipUnless(True, "won't be skipped")
```

Comment:
Are you sure this is supposed to not be executed? I got the exact opposite impression

https://docs.python.org/2/library/unittest.html#skipping-tests-and-expected-failures

So the comment message should be 'will be skipped' instead.